lib/charliecloud.py

```diff
@@ -1352,8 +1358,9 @@ def manifest_to_file(self, path, msg, digest=None):
          digest = "sha256:" + digest
       url = self._url_of("manifests", digest)
       pw = Progress_Writer(path, msg)
+      accept = "%s;q=0.5" % ",".join(TYPES_MANIFEST.values())
```

Comment:
What is the `q=0.5` magic?

Reply:
It has to do with `Accept:` weighting. I hesitate to touch it since I think I would have put it in for a good reason, but I agree it should be documented. New issue #1196.